modules/ingestor/src/graph/product/mod.rs

```diff
@@ -120,11 +120,12 @@ impl<'g> ProductContext<'g> {
 #[derive(Clone, Default, Debug)]
 pub struct ProductInformation {
     pub vendor: Option<String>,
+    pub cpe: Option<Cpe>,
 }
 
 impl ProductInformation {
     pub fn has_data(&self) -> bool {
-        self.vendor.is_some()
+        self.vendor.is_some() || self.cpe.is_some()
```

Comment:
I think idiomatic Rust might look like ` self.vendor.or(self.cpe).is_some()`

Reply:
This pattern is used for all similar code. Let's change them all in a separate PR?